src/components/swapv2/SingleTokenInfo.tsx

```diff
@@ -121,6 +120,36 @@ enum SeeStatus {
   SEE_LESS,
 }
 
+export function HowToSwap({
+  fromCurrency,
+  toCurrency,
+}: {
+  fromCurrency: Token | undefined
+  toCurrency: Token | undefined
+}) {
+  if (!fromCurrency || !toCurrency) return null
+  const symbol1 = fromCurrency.symbol
+  const symbol2 = toCurrency.symbol
+  return (
+    <Wrapper borderBottom={false}>
+      <Flex>
+        <AboutText>
+          How to swap {symbol1} to {symbol2}?
```

Comment:
You missed translations

Reply:
wrap text with <Trans> @nguyenhoaidanh
btw, why do you resolve this conversation without any comment or code update?

Reply:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/33005392/176611274-69699898-eb53-43cd-b293-5cae8de010f2.png">
because it is same as above comment

Reply:
okay.